mscore/pluginManager.cpp

```diff
@@ -67,9 +91,7 @@ void PluginManager::init()
             pluginList->setCurrentRow(0);
             pluginListItemChanged(pluginList->item(0), 0);
             }
-      connect(pluginList, SIGNAL(itemChanged(QListWidgetItem*)), SLOT(pluginLoadToggled(QListWidgetItem*)));
-      connect(pluginList, SIGNAL(currentItemChanged(QListWidgetItem*, QListWidgetItem*)),
-         SLOT(pluginListItemChanged(QListWidgetItem*, QListWidgetItem*)));
+
```

Comment:
can you remove that empty line?

Reply:
I think I've fixed that one? Not sure where it thought the line was...

Reply:
It is still in.